product_docs/docs/migration_portal/4/known_issues_notes.mdx

```diff
@@ -16,6 +17,20 @@ These are the known issues, limitations, and notes for:
 
 Migration Portal can't assess wrapped objects. If you include them in the extracted DDL, they aren't loaded into Migration Portal and aren't included in the count of objects that are assessed. If you want to assess wrapped objects and migrate them to EDB Postgres Advanced Server, include unwrapped versions of the objects in the DDL file that you upload to Migration Portal. The recommended way of doing this is to replace the wrapped versions in the Oracle database with clear-text versions before performing the schema extraction. After performing the schema extraction, you can replace the objects with the wrapped versions.
 
+### Unsupported objects
+
+Some unsupported Oracle objects are removed when Migration Portal assesses your source DDL file. These removed objects are excluded from the conversion procedure and aren't flagged as removed in the Migration Portal assessment.
+
+The following objects are removed during the schema assessment:
+
+- Objects related to materialized views, for example, backend `TABLE` or `INDEX` statements created to support `MVIEW`. 
+- All Queue-related objects.
+- Objects with nestled tables.
+- Objects related to XMLType tables.
+- Types that depend on SYSTEM schemas.
+- Indexes related to `PRIMARY KEY` and `UNIQUE` constraints.
+- Unsupported grants.
```

Comment:
@umangjtank is it possible to list the `GRANTS` that are not supported so we can provide a more helpful message to the customer?

Reply:
That list would be huge and uncertain because we are filtering them based on supported System Grants: 
CREATE DATABASE LINK
CREATE PUBLIC DATABASE LINK
DROP PUBLIC DATABASE LINK
EXEMPT ACCESS POLICY
So any System Grant coming from Oracle which is not part of this list will be marked as unsupported.